source/options.css

```
@@ -4,6 +4,13 @@
 	}
 }
 
+/* For iOS Safari */
+@media only screen and (max-width: 1000px) {
```

`width` addresses the viewport, which is ~400px in Chrome’s options (it's an iframe). This seems to be the right one https://css-tricks.com/snippets/css/media-queries-for-standard-devices/

```suggestion
@media only screen and (max-device-width: 1000px) {
```